src/core/services/theming/theming.js

```diff
@@ -153,6 +161,9 @@ function ThemingProvider($mdColorPalette) {
     extendPalette: extendPalette,
     theme: registerTheme,
 
+    registerStyles: function(styles) {
```

Comment:
@Emeegeemee Sorry, I got some different thoughts, which are not valid anymore, since `$mdColors` will come. 

Just thought again about your method, and it looks good.